core-java-modules/core-java-lang-oop-patterns/src/test/java/com/baeldung/interface_vs_abstract_class/SenderUnitTest.java

```diff
@@ -0,0 +1,21 @@
+package com.baeldung.interface_vs_abstract_class;
+
+import static org.junit.jupiter.api.Assertions.*;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+
+class SenderUnitTest {
+
+	public final static String IMAGE_FILE_PATH = "/sample_image_file_path/photo.jpg";
+
+	@Test
+	void givenImageUploaded_whenButtonClicked_thenSendImage() {
+
```

Comment:
Can we remove the empty line?

Reply:
Now removed the mentioned empty lines in both the unit tests.